tests/resources/at-commands.mjs

```diff
@@ -0,0 +1,89 @@
+
+const KNOWN_ATS = {
+  jaws: 'JAWS'
+}
+
+/* Add new commands here */
+
+const AT_COMMAND_MAP = {
+  "navigate to checkbox": {
+    reading: {
+      jaws: [
+        "Insert+Tab",
```

Comment:
It might be useful to include both the command name and the keys to execute the command. As written, I could see it being a bit confusing to testers, especially if they have not used a screen reader before? "Okay, I see that I'm supposed to press Insert+Tab, but what does that DO? And how is "up/down" different?

Also, up and down arrow keys MAY yield different results. So it might be good to break them into their own commands. Same with tab/shift tab and left/right.

Reply:
good points, @mfairchild365, I'm going to fix these things in another issue after we settle on which commands to test in #14